### DISCUSSION TITLE
Achats simultanés dans le shop

### DISCUSSION THREAD

**Discussion by Chevalier2** [Crownicles/Crownicles#574] [Idées acceptées / déjà implémentées]
Avoir la possibilité d'acheter plusieurs choses dans le shop en une seule fois sans écrire plusieurs fois la commande pour y entrer.

**Comment by niqore** [comment]
Quand est-ce que ce serait utile ? Je ne vois pas trop de cas où t'as besoin d'acheter plus de disons 3 choses

**Reply by Pietagorh** [reply]
Les gens qui achètent beaucoup d'objets d'un coup

**Comment by BastLast** [comment]
ça a été ajouté pour l'xp de guilde, pour les autres items ya moins de demande
